tests/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--no-slow", action="store_true", default=False,
```

Comment:
Pandas and Xarray call this `--skip-slow` so maybe we should call it this as well.

https://pandas.pydata.org/pandas-docs/stable/install.html#running-the-test-suite